crates/bevy_app/src/app_builder.rs

```diff
@@ -133,6 +149,32 @@ impl AppBuilder {
         self
     }
 
+    /// Adds a system that runs every time `app.update()` is called by the runner
+    ///
+    /// Systems are the main building block in the Bevy ECS app model. You can define
+    /// normal rust functions, and call `.system()` to make them be Bevy systems.
+    ///
+    /// System functions can have parameters, through which one can query and
+    /// mutate Bevy ECS states.
+    /// See [The Bevy Book](https://bevyengine.org/learn/book/introduction/) for more information.
+    ///
+    /// Systems are run in parallel, and the execution order is not deterministic.
+    /// If you want more fine-grained control for order, see [`AppBuilder::add_system_to_stage`].
```

Comment:
I think `before` and `after` are more useful general tools. Stages can be useful, but only as a last resort.

This should stay, but below comments on explicit system dependencies.